### ISSUE TITLE
Update to API changes from axI18n

### ISSUE THREAD

**Issue by jpommerening** [LaxarJS/laxar-uikit#149]
Here's an API change in Laxar that the local mocks hid from the tests: `i18n.localizeRelaxed` is gone. There is no API to supply a language tag when localizing either.

See LaxarJS/laxar#429

**Comment by jpommerening** [comment]
_(This comment will turn into **Upgrade Information**, once this ticket is completed)_

New API!

- `ui.formatter.create( type, [opts] )` is now `ui.formatter( type, [opts] )`
- `ui.parser.create( type, [opts] )` is now `ui.parser( type, [opts] )`
- `ui.i18n` has been removed
- `ui.options( [opts] )` provides default formatting and parsing options of the `ui` instance
- a localized `ui` object can be created with `ui.localized( i18n )`

Previously, users had to use `ui.i18n.momentFormatForLanguageTag` and `ui.i18n.numberFormatForLanguageTag` and then manually construct formatting options ([like ax-input-control does](https://github.com/LaxarJS/ax-input-control/blob/42943c6fc51c5fea51e924d556d29662b280033a/ax-input-control.js#L671)). These options then could be used to create formatters and parsers with `ui.formatter.create()` and `ui.parser.create()`.
Instead, UiKit now can provide formatters and parsers for a given axI18n incarnation directly, without the user having to build the options object manually. Options can still be overriden just like before. Also, a copy of the "current" options can be requested with the new `options()` method.

Here's a formal description of the new API in Typescript notation.

```ts
class AxUikitBaseOptions {
   groupingSeparator?: string,
   decimalSeparator?: string,
   dateFormat?: string,
   timeFormat?: string
}

class AxUikitFormatterOptions extends AxUikitBaseOptions {
   decimalPlaces?: integer,
   decimalTruncation?: 'FIXED' | 'BOUNDED' | 'NONE'
}

class AxUikitParserOptions extends AxUikitBaseOptions {
   dateFormatFallbacks?: array,
   dateTwoDigitYearWrap?: integer,
   timeFormatFallbacks?: array
}

class AxUikit {
   options( opts?: AxUikitFormatterOptions | AxUikitParserOptions ): Object;
   formatter( type: string, opts?: AxUikitFormatterOptions ): Function;
   parser( type: string, opts?: AxUikitParserOptions ): Function;
   localized( i18n: AxI18n ): AxUikit;
}
```

/cc @alex3683, @x1B

**Comment by jpommerening** [comment]
Published w/ in `v2.0.0-alpha.4`
